src/core/inlines.js

```diff
@@ -22,6 +22,72 @@ import { renderInlineCitation } from "./render-biblio";
 export const name = "core/inlines";
 export const rfc2119Usage = {};
 
+/**
+ * @param {string} matched
+ */
+function inlineRFC2119Matches(matched) {
+  const normalize = matched.split(/\s+/).join(" ");
+  const nodeElement = hyperHTML`<em class="rfc2119" title="${normalize}">${normalize}</em>`;
+  // remember which ones were used
+  rfc2119Usage[normalize] = true;
+  return /** @type {HTMLElement} */ nodeElement;
+}
+
+/**
+ * @param {string} matched
+ */
+function inlineXrefMatches(matched) {
+  // slices "{{{" at the beginning and "}}}" at the end
+  const ref = matched.slice(3, -3).trim();
+  return /** @type {HTMLElement} */ ref.startsWith("\\")
```

Comment:
```suggestion
  return ref.startsWith("\\")
```

Doesn't look needed, it should be automatically inferred as `Node`.